### PR TITLE
Weston: Fix authentication error for egl clients

### PR DESCRIPTION
**Pull request by harishkrupo**
Signed-off-by: Harish Krupo <harishkrupo@gmail.com>

### CODE REVIEW

File: os/linux/weston/plugin/compositor-iahwc.c [hunk 1]
```diff
@@ -1672,8 +1671,9 @@ static void session_notify(struct wl_listener *listener, void *data) {
   }
 }
 
-static void planes_binding(struct weston_keyboard *keyboard, uint32_t time,
-                           uint32_t key, void *data) {
+static void
+planes_binding(struct weston_keyboard *keyboard, const struct timespec* time,
```
Comment (kalyankondapally) [inline os/linux/weston/plugin/compositor-iahwc.c]: can u separate indentation fixes to a separate patch

File: os/linux/weston/plugin/compositor-iahwc.c [hunk 2]
```diff
@@ -1705,11 +1706,10 @@ static struct iahwc_backend *iahwc_backend_create(
   iahwc_module_t *iahwc_module;
   iahwc_device_t *iahwc_device;
 
-  /* const char* device = "/dev/dri/renderD128"; */
-  const char *device = "/dev/dri/card0";
+  const char* device = "/dev/dri/renderD128";
```
Comment (kalyankondapally) [inline os/linux/weston/plugin/compositor-iahwc.c]: Can you explain in commit message the problem and how is it fixed.

File: os/linux/weston/plugin/compositor-iahwc.c (hunk 1 above)
Comment (kalyankondapally) [reply os/linux/weston/plugin/compositor-iahwc.c]: Perhaps fix any other indentation issues same time

Comment (harishkrupo) [reply os/linux/weston/plugin/compositor-iahwc.c]: ok, sure.

File: os/linux/weston/plugin/compositor-iahwc.c (hunk 2 above)
Comment (harishkrupo) [reply os/linux/weston/plugin/compositor-iahwc.c]: Okay.

**Comment by kalyankondapally** [comment]
still issue:
"make[1]: *** No rule to make target `../iahwc/os/linux/weston/plugin/compositor-iahwc.c', needed by `../iahwc/os/linux/weston/plugin/iahwc_backend_la-compositor-iahwc.lo'.  Stop.
make[1]: *** Waiting for unfinished jobs....
  CC       libweston/gl_renderer_la-vertex-clipping.lo
make[1]: Leaving directory `/home/travis/build/intel/weston'"

**Comment by harishkrupo** [comment]
Yes, the issue is because of the compositor-iahwc.c path is hardcoded to ../iahwc/..../compositor-iahwc.c.
Previously the build_scipt.sh would modify the patch to the right path and apply it. This way weston would correctly find the file. Now that it is hardcoded we will have to find another way out.

**Comment by kalyankondapally** [comment]
https://github.com/intel/IA-Hardware-Composer/pull/316 can u try with these changes, still same error :(

**Comment by harishkrupo** [comment]
Same error? Are you referring to travis or the authentication issue?

**Comment by harishkrupo** [comment]
@kalyankondapally travis fixed

**Comment by kalyankondapally** [comment]
thx merged
